core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala

```diff
@@ -978,7 +982,11 @@ class WorkflowSubmissionSpec(_system: ActorSystem)
     val mockExecCluster = MockShardedExecutionServiceCluster.fromDAO(new MockExecutionServiceDAO(), slickDataSource)
     val workspaceSettingRepository = mock[WorkspaceSettingRepository]
     when(workspaceSettingRepository.getWorkspaceSettings(UUID.fromString(testData.workspace.workspaceId))).thenReturn(
-      Future.successful(List(UseCromwellGcpBatchBackendSetting(UseCromwellGcpBatchBackendConfig(true))))
+      Future.successful(
+        List(GcpBucketSoftDeleteSetting(GcpBucketSoftDeleteConfig(604800)),
+             UseCromwellGcpBatchBackendSetting(UseCromwellGcpBatchBackendConfig(true))
+        )
+      )
```

Comment:
Why did this expectation change? How was the test passing before?

Reply:
This is to cover the case where `UseCromwellGcpBatchBackendSetting` is not the only setting that exists for the workspace.

Reply:
Ah - my apologies, I misread this; I thought it was part of the assertion, not part of the mock. My bad!